pkg/apiclient/workflow/workflow.proto

```diff
@@ -43,7 +43,7 @@ message WorkflowResubmitRequest {
 message WorkflowRetryRequest {
     string name = 1;
     string namespace = 2;
-    bool restartSuccesful = 3;
```

Comment:
Is this a breaking change? Has this gone GA yet?

Reply:
It is breaking, but it's part of v2.8 and hasn't gone GA yet.

Reply:
👍